llama_hub/airbyte_cdk/base.py

```diff
@@ -0,0 +1,54 @@
+import json
+from typing import Any, Callable, Iterator, List, Mapping, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from airbyte_protocol.models.airbyte_protocol import AirbyteRecordMessage
```

Comment:
so we should figure out a better test system on our side, but currently tests are failing because we assume that third-party imports are lazy imports.

i know specifically for AirbyteRecordMessage it's used in the `RecordHandler` type used to type `record_handler`. For this do you think we could type `record_handler` as `Any` first, and then within `__init__` lazy import `AirbyteRecordMessage`, import `RecordHandler`, and do an `isinstance()` check on `record_handler`? 

A bit hacky but will at least allow tests to pass. thanks